### DISCUSSION TITLE
How to disable the close button in Firefox

### DISCUSSION THREAD

**Discussion by mahaveergurjar** [prasanthrangan/hyprdots#1119] [Q&A]
How to disable the close button in Firefox 
![240319_18h49m19s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/57872034/da088304-306c-4f33-bde4-dd5e9abae237)

**Comment by kRHYME7** [comment]
``` Customize toolbar ``` 

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/bf0d5a6a-8f0e-4734-a1c4-d5cc08c82c33)


![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/c8b8928d-701b-4bdb-adfd-583f2d8194cc)

**Comment by mahaveergurjar** [comment]
Okay Thanks
